doc/backend/02obtain_access_tokens.md

```diff
@@ -0,0 +1,60 @@
+# How to obtain access tokens?
+
+In this guide it will be explained how to obtain access tokens and make requests to the backend while developing.
+
+Be careful not to leak your username/password or access_token during the following steps.
+
+## Using [Postman](https://www.postman.com/)
+
+- Open Postman.
+- Create a new request.
+  - Enter the URL <http://localhost:8080/api/config> and execute the request.  
+    This should now execute without errors.
+  - Enter <http://localhost:8080/api/seeds>.  
+    You should get a status code `401 Unauthorized`.
+- Click the `Authorization` tab and select `OAuth 2.0`.
+  - Chose `Grant Type` `Authorization Code`.
+  - Set `Callback URL` to <http://localhost:5173/>.
+  - Set `Auth URL` to <https://auth.permaplant.net/realms/PermaplanT/protocol/openid-connect/auth>.
+  - Set `Access Token URL` to <https://auth.permaplant.net/realms/PermaplanT/protocol/openid-connect/token>.
+  - Set `Client ID` to `localhost`.
+  - Click `Get New Access Token` at the bottom.
+    - Postman should now open a browser window.
+    - Enter your credentials.
+    - There should now be a window displaying `Authentication complete`.
+    - Once it closes click `Use Token`.
+- You should now be able to execute the request to <http://localhost:8080/api/seeds> in Postman.
+
+A more in depth explanation about Postman and `OAuth 2.0` can be found in the [Postman documentation](https://learning.postman.com/docs/sending-requests/authorization/oauth-20/).
+
+## Using Curl
+
+The following request should work without problems:  
+`curl 'http://localhost:8080/api/config'`
+
+The following request should fail:  
+`curl 'http://localhost:8080/api/seeds'`
```

Comment:
Does this actually fail for you? I simply don't get any response but `curl` executes successfully. (Tried on master).